pulp_rpm/app/migrations/0004_optimized_sync.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.10 on 2020-03-04 20:09
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0022_rename_last_version'),
+        ('rpm', '0003_DATA_incorrect_json'),
```

Comment:
You need to regenerate the migration. A new migration was added since you last generated this one.